### DISCUSSION TITLE
How to update all columns on conflict

### DISCUSSION THREAD

**Discussion by rsmidt** [go-jet/jet#237] [Q&A] [answered]
I'm trying to implement a basic upsert where I want to update all fields as soon as I hit a constraint, e.g.

```go
Users.
	INSERT(Users.AllColumns).
	MODEL(user).
	ON_CONFLICT(Users.ID).
	DO_UPDATE(
		SET(Users.AllColumns.SET(Users.EXCLUDED.AllColumns)),
	)
```

Unfortunately, this doesn't compile. Am I doing something wrong here? Is there maybe a better way to achieve the upsert?

**Comment by go-jet** [comment, marked as answer]
Hi @rsmidt ,
No currently it's not possible to upsert using `ColumnList` on the right side. On the right side only `ROW` can appear. You'll have to set each column manually.   
I'm gonna move this to an issue. It might be worth to add a support in the future.

**Reply by rsmidt** [reply]
Thanks! Then, for now, I will specify them manually.
